qcodes_contrib_drivers/drivers/QDevil/QDAC2.py

```diff
@@ -2170,3 +2171,10 @@ def _set_up_manual_triggers(self) -> None:
     def _set_up_simple_functions(self) -> None:
         self.add_function('reset', call_cmd='*rst')
         self.add_function('abort', call_cmd='abor')
+
+    def _check_instrument_name(self, name: str) -> None:
```

Comment:
stricktly qcodes does not prescribe this requirement for instrument names. but perhaps it should? could you explain when did it become a problem for qdac?

Reply:
The `doNd` functions break when the instrument name is not `isidentifier()`